src/main/scala/Config.scala

```diff
@@ -0,0 +1,8 @@
+package io.scalac.wtf.domain
+
+import slick.jdbc.JdbcBackend.Database
+
+import scala.concurrent.ExecutionContext
+
+case class Config(ec: ExecutionContext,
+                  db: Database)
```

Comment:
Question: it's a good practice to have separate `ExecutionContext`s for different tasks ... what if I wanted to use many? (not a call to action)